contracts/src/v0.8/functions/dev/1_0_0/interfaces/IFunctionsRequest.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
+
+interface IFunctionsRequest {
```

Comment:
Do we need a separate file because it's not used by both Router and Billing?

Reply:
I agree with this.  I do prefer a single shared struct.